libs/payments/stripe/src/lib/subscription-item.factory.ts

```diff
@@ -0,0 +1,50 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { faker } from '@faker-js/faker';
+import { Stripe } from 'stripe';
+import { PriceFactory } from './price.factory';
+
+export const SubscriptionItemFactory = (
+  subscriptionId: string,
+  productId: string,
+  planId: string,
```

Comment:
Factory really should only take an overrides argument, and the rest should be auto-generated.